extend.php

```diff
@@ -85,19 +86,24 @@
         ->addSortField('bestAnswerCount'),
 
     (new Extend\Settings())
+        ->default('fof-best-answer.schedule_on_one_server', false)
+        ->default('fof-best-answer.stop_overnight', false)
+        ->default('fof-best-answer.store_log_output', false)
+        ->default('fof-best-answer.enabled-tags', '[]')
+        ->default('fof-best-answer.search.solution_search', true)
```

Comment:
I think it would probably better to make this false. Although we, as maintainers, expect people to read the changelog, when running a generic `composer update` it is fairly vague to see another search source pop up. Please consider this 🙏